### ISSUE TITLE
Update `uniqueValues` in twisted/internet/_sslverify.py to use `set` instead of `dict` contortions

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5749]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#5749|
|Type|enhancement|
|Created|2012-07-08 22:13:47Z|

This legacy code exists due to the late introduction of `set` into Python:

```python
       def uniqueValues(mapping):
            return dict.fromkeys(mapping.itervalues()).keys()
```

It's simpler (and probably more efficient) to express this idea using `set`.

Attachments:

* [5749.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/76a/76a93eb97ffb6016b155ba6c930709841aaadbfb/a0d4228b78e2c03d3a91e2c7621ad98b3b42811c.patch) (548 bytes) - added by nry on 2012-07-17 12:06:38Z - uses a set to return unique values

<details><summary>Searchable metadata</summary>

```
trac-id__5749 5749
type__enhancement enhancement
reporter__exarkun exarkun
priority__normal normal
milestone__ 
branch__ 
branch_author__nry nry
status__closed closed
resolution__fixed fixed
component__core core
keywords__easy easy
time__1341785627000000 1341785627000000
changetime__1353564824000000 1353564824000000
version__None None
owner__exarkun exarkun
cc__nry
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="nry's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|nry commented|
|-|-|

Uses a set to return unique values from a dictionary.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @exarkun|
|-|-|

Thanks.  This looks good.  I'll apply it.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [34889]) Replace historic dict.fromkeys().keys() with set()

Author: nry
Reviewer: exarkun
Fixes: [#5749](https://github.com/twisted/twisted/issues/5749)
